README.md

```diff
@@ -12,6 +12,9 @@ What all can you make:
 - Hero section template
 - Navigation Bar
 - Projects Section
+- Quiz App
```

Comment:
Don't update this everytime you add a new template

Reply:
Ok, I am sorry